R/print.R

```diff
@@ -9,28 +9,21 @@ format.formula_blueprint <- function(x, ...) "Formula"
 
 #' @export
 print.hardhat_blueprint <- function(x, ...) {
-  cat_line("{format(x)} blueprint:")
-  cat_line("\n")
-  cat_line("# Predictors: {n_blueprint_predictors(x)}")
-  cat_line("  # Outcomes: {n_blueprint_outcomes(x)}")
-  cat_line("   Intercept: {x$intercept}")
-  cat_line("Novel Levels: {x$allow_novel_levels}")
-  cat_line(" Composition: {x$composition}")
```

Comment:
cli generally strips the leading whitespace so getting these to stay aligned at the `:` would have required (quite a bit) more crafting so I went with keeping it straightforward and aligning things at the start of the line.